app/models/mixins/archived_mixin.rb

```diff
@@ -2,11 +2,14 @@ module ArchivedMixin
   extend ActiveSupport::Concern
 
   included do
+    scope :deleted, -> { where(:deleted => true) }
+    scope :not_deleted, -> { where(:deleted => false) }
```

Comment:
we agreed on :active, :archived scopes. The all doesn't need a scope since we will just call a model.all. We could add it later though, if we would plan something like :default_scope, that would affect model.all calls

The scopes on the ems association are there, because it will then work as before. So it would be more error prone to change the results of those call.